### DISCUSSION TITLE
idk won't start

### DISCUSSION THREAD

**Discussion by GogoFC** [nextcloud/all-in-one#2155] [Questions] [answered]
After I login on the initial install this happens.

![Screenshot from 2023-03-15 02-23-57](https://user-images.githubusercontent.com/9360410/225181616-47a3b4f0-1f7c-4492-ba4b-4e193343841b.png)

**Comment by szaimen** [comment, marked as answer]
Hi, did you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md?

**Reply by GogoFC** [reply]
No I followed the README file, number 2. Proxy instructions would have been next had the README instructions worked for me.
